lib/service/skills/OnetOccupationsData.dart

```diff
@@ -0,0 +1,114 @@
+import 'dart:convert';
+
+import 'package:dio/dio.dart';
+import 'package:flutter/foundation.dart';
+import 'package:illinois/model/occupation/Occupation.dart';
+import 'package:illinois/service/Config.dart';
+import 'package:illinois/service/skills/IOccupationsData.dart';
+
+class OnetOccupationsData implements IOccupationsData {
```

Comment:
I would suggest combining this class with `OccupationsService` to keep it consistent with our other services. Also, in order to start your service on app startup, you will need to add `OccupationsService()` to the list passed to `illinois.Services().create` in `mainImpl.dart`.